docs/README.md

```diff
@@ -0,0 +1,48 @@
+
+# PCUI docs guide
+
+### Updating component pages (Optional if you’ve added a new component)
+
+Open `./docs/create-component-pages.js`
+
+Add your new component to the components array (including the category and component name).
+
+`cd ./docs`
+
+`node create-component-pages.js`
+
+Commit any changes to the pcui repo.
+
+### Testing docs locally
+
+If you haven't cloned the [playcanvas.github.io](https://github.com/playcanvas/playcanvas.github.io) repo, install it locally to a projects folder which will now be referenced as `<projects_folder>`.
+
+In the puci directory run:
+
+`npm run docs:local`
+
+`rm -rf ~/<projects_folder>/playcanvas.github.io/pcui/`
+
+`cp -r ./docs/_site/ ~/<projects_folder>/playcanvas.github.io/pcui/`
+
+`cd ~/<projects_folder>/playcanvas.github.io`
+
+`python -m SimpleHTTPServer 4000`
```

Comment:
maybe we shouldn't be adding python stuff in here.. Node has an `http-server` module that can start servers

Reply:
Very much +1 on that. 👍